### PR TITLE
Add support nrf5 temp sensor

### PR DESCRIPTION
**Pull request by ojousima**

### CODE REVIEW

File: src/app_sensor.h [hunk 1]
```diff
@@ -227,6 +228,23 @@ void m_sensors_init (void); //!< Give Ceedling a handle to initialize structs.
...
 #endif
 
+#if APP_SENSOR_ENVIRONMENTAL_MCU_ENABLED
+#define APP_SENSOR_ENVIRONMENTAL_MCU_DEFAULT_CFG  \
+  {                                               \
+    .sensor = {0},                                \
+    .init = &ri_environmental_mcu_init,           \
+    .configuration = {0},                         \
+    .nvm_file = APP_FLASH_SENSOR_FILE,            \
+    .nvm_record = APP_FLASH_SENSOR_NTC_RECORD,    \
```
Comment (ojousima) [inline src/app_sensor.h:238]: APP_FLASH_SENSOR_NTC_RECORD -> APP_FLASH_SENSOR_MCU_RECORD

#define in app_config.h

**Review by ojousima — COMMENTED** [review]
Define the flash record

File: src/app_sensor.c [hunk 2]
```diff
@@ -832,7 +836,7 @@ static rd_status_t app_sensor_log_read (const ri_comm_xfer_fp_t reply_fp,
             {
                 err_code |= app_sensor_send_eof (reply_fp, raw_message);
                 char msg[128];
-                snprintf (msg, sizeof (msg), "Logged data sent: %lu elements\r\n", sent_elements);
+                snprintf (msg, sizeof (msg), "Logged data sent: %d elements\r\n", sent_elements);
```
Comment (ojousima) [inline src/app_sensor.c]: It was %lu on purpose

File: src/app_sensor.c [hunk 3]
```diff
@@ -832,7 +836,7 @@ static rd_status_t app_sensor_log_read (const ri_comm_xfer_fp_t reply_fp,
             {
                 err_code |= app_sensor_send_eof (reply_fp, raw_message);
                 char msg[128];
-                snprintf (msg, sizeof (msg), "Logged data sent: %lu elements\r\n", sent_elements);
+                snprintf (msg, sizeof (msg), "Logged data sent: %ld elements\r\n", sent_elements);
```
Comment (ojousima) [inline src/app_sensor.c:839]: %lu, not %ld

File: src/app_sensor.h (hunk 1 above)
Comment (niksiboxi) [reply src/app_sensor.h]: Defined as `#define APP_FLASH_SENSOR_ENVI_RECORD     (0x52U)`

**Pull request merged**